### ISSUE TITLE
Picked cell rendering blue before pink

### ISSUE THREAD

**Issue by rodrigomologni** [pyvista/pyvista#916]
Hi, Team! 

I'm using `enable_cell_picking` with the default parameters. When the callback function is time consuming, the picked cell is rendered blue before pink. See the screenshots below.

![Untitled](https://user-images.githubusercontent.com/1608652/94878702-86b8ca80-0434-11eb-94e3-1d54071a185a.png)

A demo for test...

```python
import time
import pyvista as pv

def cell_picking(subgrid):
    print(subgrid)
    time.sleep(2)

grid = pv.UniformGrid((30, 30, 10))

plotter = pv.Plotter()
plotter.add_mesh(grid, show_edges=True)
plotter.enable_cell_picking(grid, callback=cell_picking, through=False)
plotter.show()
```

**Comment by akaszynski** [comment]
Thanks for pointing this out!  I've fixed this in #919, and it the fix will be out in the next release.

**Comment by rodrigomologni** [comment]
@akaszynski Ohhh... Sorry! Great news! ✌️
